crates/net/network/src/transactions.rs

```diff
@@ -580,10 +579,11 @@ where
                     this.on_good_import(hash);
                 }
                 Err(err) => {
-                    // if we're syncing and the transaction is bad we ignore it, otherwise we
-                    // penalize the peer that sent the bad transaction with the assumption that the
-                    // peer should have known that this transaction is bad. (e.g. consensus rules)
-                    if err.is_bad_transaction() && !this.network.is_syncing() {
+                    // if we're initially syncing and the transaction is bad we ignore it, otherwise
+                    // we penalize the peer that sent the bad transaction with
+                    // the assumption that the peer should have known that this
+                    // transaction is bad. (e.g. consensus rules)
+                    if err.is_bad_transaction() && !this.network.is_initially_syncing() {
```

Comment:
I changed this to currently syncing

Reply:
yeah I think we need this for every sync, good catch thanks